apps/src/applab/applab.js

```diff
@@ -1315,7 +1329,38 @@ Applab.onPropertyChange = function(element, name, value) {
       // element.checked represents the current state, the attribute represents
       // the serialized state
       element.checked = value;
-      element.setAttribute('checked', value ? 'checked' : null);
+      if (value) {
+        element.setAttribute('checked', 'checked');
+      } else {
+        element.removeAttribute('checked');
+      }
+      break;
+    case 'options':
+      // value you should be an array of options in this case
```

Comment:
change to "value should be..." ? alternately you could throw if value is not an Array.